math/src/main/java/smile/math/matrix/Matrix.java

```diff
@@ -456,6 +456,32 @@ default SVD svd(int k, double kappa, int maxIter) {
             return new SVD(U, V, s);
         }
     }
+    
+    /**
```

Comment:
I have redesigned the matrix library. When it is ready, I will merge it to arima branch.

Reply:
I have merged arima with master. Please update. Thanks

Reply:
@haifengl I updated the PR. However, with new change around `Matrix` and its related methods like `Cholesky`, the `ARIMATest` which could pass before now break. I tried several ways but not working as expected. I thought the new change to `Matrix` be transparent to the caller but it require mkl libraray now as well. Before it was some pure Java implementation like `JMatrix`.